README.md

```diff
@@ -8,19 +8,42 @@
 
 # Installation 
 
-1. `npm install screepsmod-auth` in your server folder.
-2. Thats it!
+1a. If you are using the in-game private server from Steam, then subscribe the Steam workshop  https://steamcommunity.com/sharedfiles/filedetails/?id=800390576
 
-# Usage
+1b. If you are using the screeps standalone private server, run `npm install screepsmod-auth` folder, and modify mods.json and add a new line in your mods list pointing to this mod like so:
 
-## Steam Console method
-1. Open the steam screeps client
-2. In console, run `setPassword('YourDesiredPassword')`
-3. Now you should be able to login via API
+```
+  "mods": [
+    "node_modules\\screepsmod-auth\\index.js"
+  ],
+```
+
+2. Relaunch your screeps private server.
+
+3. Follow the 
+[CLI steps](#screeps-server-command-line-interface-cli-method) below to setup your username and password 
+
+# Setup
+
+## Screeps Server Command Line Interface (CLI) method
+1. Connect to your screeps private server once to create your account.  You don't need to set an email.  Just username and password is important.
+
+2. Run the server's command line interface (CLI) either in Steam, or if your running a standalone server as described here https://github.com/screeps/screeps#command-line-interface-cli
 
-## Web Form Method
-1. Open the steam client at least once (Required to create initial account)
-2. Goto http://yourServerHostOrIP:21025/authmod/password/
+3. In server's CLI, run 
+
+```
+setPassword('yourUserName', 'yourPassword')
+```
+Note: that setPassword says to pass in the email.  Don't do this.  Instead give it your username.
+
+4. If you get back an error that setPassword isn't defined, then its likely your mod isn't installed.  See the mod.json set above to fix it.  If setPassword returns {} then that's also a failure as your user name wasn't found.  If setPassword returns { modified: 1 }, then it succceded.
+
+5. Now you should be able to login 
+
+## Web Form Method 
+1. Connect to your screeps private server once to create your account. 
+2. Send a POST call to http://yourServerHostOrIP:21025/authmod/password/
```

Comment:
This is still not valid, as mentioned in the previous comments, this will not work. Needs both the steam auth and the password data or it will not apply.